### PR TITLE
[bug] Add Meroxa-Account-UUID to API Requests

### PR DESCRIPTION
**Pull request by ericcheatham**
Part of https://github.com/meroxa/turbine-project/issues/277

Meroxa-py client now requires `meroxa_account_uuid` which will then be set as a header value for  `Meroxa-Acconut-UUID`

### CODE REVIEW

File: src/meroxa/types.py [hunk 1]
```diff
@@ -23,13 +23,6 @@ def __init__(self, *args, **kwargs):
         ...
 
 
-class ClientOptions:
```
Comment (ericcheatham) [inline src/meroxa/types.py:26]: This was not being used anywhere but was the cause of some weird import shadowing issues in Turbine-py

File: src/meroxa/applications.py [hunk 2]
```diff
@@ -16,23 +16,23 @@ def __init__(
         uuid: str,
         name: str,
         language: str,
-        git_sha: str,
         status: dict,
         pipeline: dict,
         created_at: str,
         updated_at: str,
+        git_sha: str = None,
```
Comment (ericcheatham) [inline src/meroxa/applications.py:23]: `git_sha` can be omitted in in the response from the API. In the event that this happens we would get an error as we attempt to unpack the JSON response into an object that expects a particular named param to exist.

Comment (ericcheatham) [reply src/meroxa/applications.py:23]: This also addresses the following error:
```
Error: __init__() got an unexpected keyword argument 'created_at'
```

File: src/meroxa/types.py (hunk 1 above)
Comment (jayjayjpg) [reply src/meroxa/types.py:26]: Nice cleanup! :sparkles:

File: src/meroxa/client.py [hunk 3]
```diff
@@ -23,7 +28,10 @@ def __init__(
         if session is None:
             session = aiohttp.ClientSession(
                 base_url=api_route,
-                headers={"Authorization": "Bearer {}".format(auth)},
+                headers={
+                    "Authorization": "Bearer {}".format(auth),
+                    "Meroxa-Account-UUID": meroxa_account_uuid,
```
Comment (jayjayjpg) [inline src/meroxa/client.py:33]: This reads great!

**Review by jayjayjpg — APPROVED** [review]
This reads great, thank you for the super quick update! :sparkles:

File: src/meroxa/client.py (hunk 3 above)
Comment (janelletavares) [reply src/meroxa/client.py:33]: i think there's a chance that meroxa_account_uuid is an empty string, so i think we should check that it's not empty before setting it.  this maybe the case if there's an older CLI but a new py app.

**Review by janelletavares — CHANGES_REQUESTED** [review]

**Review by janelletavares — APPROVED** [review]

**Pull request merged**